models/app.models.js

```diff
@@ -5,3 +5,14 @@ exports.selectTopics = (req, res) => {
     return rows;
   });
 };
+
+exports.SelectArticlesByID = (id) => {
+  return db
+    .query(
+      `SELECT author, title, article_id, body, topic, created_at, votes, article_img_url FROM articles WHERE article_id = $1`,
+      [id]
+    )
+    .then(({ rows }) => {
+      return rows[0];
+    });
+};
```

Comment:
Good job with this so far, if you add the 404 test you may want to check if rows is empty here 👀